### PR TITLE
Test Pull Request

### PR DESCRIPTION
**Pull request by rasnaut**
Task: #2

### CODE REVIEW

File: ClientServerApp.sln [hunk 1]
```diff
@@ -9,6 +9,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Client", "Client\Client.vcx
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Client2", "Client2\Client2.vcxproj", "{27FABDEB-E38B-4312-87B1-B5FFD8668660}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "UIClient", "UIClient\UIClient.vcxproj", "{13E7E5F1-76FE-47FE-A974-3BD8CFC36FC4}"
```
Comment (rasnaut) [inline ClientServerApp.sln:12]: Problem

File: Server/Server.cpp [hunk 2]
```diff
@@ -3,26 +3,31 @@
 
 #include <iostream>
 #include <vector>
+#include <unordered_map>
```
Comment (rasnaut) [inline Server/Server.cpp:6]: Block

**Review by rasnaut — COMMENTED** [review]
please change according comments

File: ClientServerApp.sln [hunk 3]
```diff
@@ -9,6 +9,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Client", "Client\Client.vcx
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Client2", "Client2\Client2.vcxproj", "{27FABDEB-E38B-4312-87B1-B5FFD8668660}"
```
Comment (sidhtc510) [inline ClientServerApp.sln:10]: good

**Review by sidhtc510 — CHANGES_REQUESTED** [review]
add some changes

File: ClientServerApp.sln (hunk 3 above)
Comment (rasnaut) [reply ClientServerApp.sln:10]: observed

**Review by sidhtc510 — APPROVED** [review]

**Pull request merged**